rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -1614,6 +1694,10 @@ public ResponseEntity<HalResource> addAttachmentToProject(
         }
 
         project.addToAttachments(attachment);
+        if (!restControllerHelper.isWriteActionAllowed(project, sw360User) && comment == null) {
+            return new ResponseEntity(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?